### PR TITLE
Add New York file

### LINKED ISSUE #5: XuboS tracks progress on GH Flow
- [ ] create a feature branch
- [ ] add some commits
- [ ] open a pull request
- [ ] collaborate and add additional commits
- [ ] merge my pull request with 1 👍 

/play greatjob

### PR DESCRIPTION
**Pull request by XuboS**
This PR adds the New York file to our project. This PR closes #5 

Some ❓s:
1. if you've been to New York before, what did you like? what did you not like?

### CODE REVIEW

File: XuboS-newyork.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# New York
+
+### Restaurants
+  - <a href='http://www.purethaicookhouse.com/about'>Pure Thai Cookhouse</a>
+  - <a href='http://tottoramen.com'>Totto Ramen</a>
+  
+### Local Attractions
+  - Statue of Liberty
```
Comment (ahu2012) [inline XuboS-newyork.md:8]: where is this?

Comment (alanzhong) [inline XuboS-newyork.md:8]: :O

File: XuboS-newyork.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# New York
+
+### Restaurants
+  - <a href='http://www.purethaicookhouse.com/about'>Pure Thai Cookhouse</a>
+  - <a href='http://tottoramen.com'>Totto Ramen</a>
```
Comment (hollenberry) [inline XuboS-newyork.md]: I love the href additions! Excellent! Do you recommend Totto over Ippudo?

**Comment by mmanfred** [comment]
Thanks for recommendations

**Comment by alanzhong** [comment]
:ship:

**Pull request merged**